### PR TITLE
Task/v 9

### PR DESCRIPTION
**Pull request by XavierVal**
ADD CI SUPPORT

### CODE REVIEW

File: tests/steps/happy_path.py [hunk 1]
```diff
@@ -274,8 +337,34 @@ def step_impl(context):
...
+    :type SERVICE_ADMIN: str
+    :type SERVICE_PWD: str
+    """
+    pass
+
+    context.service_admin = SERVICE_ADMIN
+    context.service_password = SERVICE_PWD
+
+    context.subservices = orc_get_subservices(context, context.service_id)
+    # print ("\n #>> SUB_Services availables: {} \n".format(context.subservices))
```
Comment (dmoranj) [inline tests/steps/happy_path.py]: Why is this line commented out?

File: tests/steps/smoke_steps.py [hunk 2]
```diff
@@ -213,7 +216,35 @@ def step_impl(context, version):
...
+    returned_version = component_version_check(context, component=comp)
+
+    # compare the version with the expected one
+    eq_(returned_version, version,
+        '[{}] Not the correct version: found({}) expected({})'.format(comp, returned_version, version))
+
+    __logger__.debug("{} Version: {}".format(comp, returned_version))
+
+
+@then(u'the returned version from "PEP" should match the "(?P<version>.+)"')
```
Comment (dmoranj) [inline tests/steps/smoke_steps.py]: It seems to me that all the "version steps" could be collapsed into a single one with parametrized component name (they seem to differ only in the component name text).

File: tests/steps/happy_path.py (hunk 1 above)
Comment (XavierVal) [inline tests/steps/happy_path.py]: Just for nothing, removed
Fixed in 66909e0

File: tests/steps/smoke_steps.py (hunk 2 above)
Comment (XavierVal) [inline tests/steps/smoke_steps.py]: The platform components returns the value of the version in a different formats that require a special treatment to extract it so we need the different version steps. But you are right that the Node components version checkers do the same... unified 

Fixed in 84fb2d8

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**